easybuild/easyconfigs/s/slurm-drmaa/slurm-drmaa-1.1.3-GCCcore-12.2.0.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'slurm-drmaa'
+version = '1.1.3'
+
+homepage = "https://github.com/natefoo/slurm-drmaa"
+description = """DRMAA for Slurm Workload Manager (Slurm) is an implementation
+ of Open Grid Forum Distributed Resource Management Application API (DRMAA)
+ version 1 for submission and control of jobs to Slurm. Using DRMAA,
+ grid applications builders, portal developers and ISVs can use the same
+ high-level API to link their software with different cluster/resource management systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/natefoo/slurm-drmaa/releases/download/%(version)s/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['145eb1bc3f0f0aac936505eb29ac501dee32228f70781d41df4f032939b9c3ba']
+
+builddependencies = [
+    ('binutils', '2.39'),
+]
+
+sanity_check_paths = {
+    'files': [
+        'include/drmaa.h',
+        'lib/libdrmaa.a',
+        'lib/libdrmaa.so',
+        'bin/drmaa-run',
+    ],
+    'dirs': ['etc']
+}
```

Order entries alphabetically, no need to extend this across multiple lines, and use `SHLIB_EXT`:

```suggestion
sanity_check_paths = {
    'files': ['bin/drmaa-run', 'include/drmaa.h', 'lib/libdrmaa.a', 'lib/libdrmaa.%s' % SHLIB_EXT],
    'dirs': ['etc'],
}